### PR TITLE
PAT-92 Add view capability and edit functionality for actions

### PR DESCRIPTION
**Pull request by yasinmustafa**

### CODE REVIEW

File: server/config/ActionStatus.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 const NOT_STARTED = 'Not started'
 const IN_PROGRESS = 'In progress'
 const PAUSED = 'Paused'
+const PARTLY_COMPLETED = 'Partly completed'
```
Comment (lbennett-moj) [inline server/config/ActionStatus.js]: Out of interest - what is the difference between IN_PROGRESS and PARTLY_COMPLETED? Do we need both?

**Comment by lbennett-moj** [comment]
Is an integration test planned to cover this?

File: server/config/ActionStatus.js (hunk 1 above)
Comment (yasinmustafa) [reply server/config/ActionStatus.js]: Good question... afraid I dont know the answer. The status came directly from the AC's and I didn't ask that question!

**Comment by yasinmustafa** [comment]
> Is an integration test planned to cover this?

Discussed with Tim and decided to wait until caseload which may throw a spanner in the works

**Pull request merged**